Dockerfile

```diff
@@ -1,20 +1,17 @@
 FROM docker.io/centos:centos7
 
-# TODO(iurygregory): remove epel-release and  other packages necessary for the ironic_prometheus_exporter when we have a package for it
 RUN yum install -y python-requests && \
     curl https://raw.githubusercontent.com/openstack/tripleo-repos/master/tripleo_repos/main.py | python - current-tripleo && \
     yum update -y && \
-    yum install -y epel-release python-pip python-devel gcc openstack-ironic-api openstack-ironic-conductor crudini \
+    yum install -y uwsgi openstack-ironic-api openstack-ironic-conductor crudini \
```

Comment:
My CentOS system also has uwsgi-plugin-python, do we need it?

Reply:
Yeah it's required